custom/icds_reports/templates/icds_reports/dashboard.html

```diff
@@ -95,7 +95,6 @@
 
         <script src="{% static 'js/directives/awc-opened-yesterday/awc-opened-yesterday.directive.js' %}"></script>
         <script src="{% static 'js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js' %}"></script>
-        <script src="{% static 'js/directives/awc-reports/awc-reports.directive.js' %}"></script>
```

Comment:
this commit also should have updated `mocha.html` but I missed that until the end.